ckanext/budgetviz/groupbarchart_munc/theme/templates/groupbarchart_view.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'dataviewer/base.html' %}
+{% block page %}
+ <div class="infocard">
+            <div class="row">
+                <div class="span7">
+                    <div class="viz-header">
+                        <span id="viz-header"></span>
+                </div>
+            </div>
+            <div class="span5">
+                <div class="scheme-select"></div>
+                
+            </div>
+        </div>
+        <hr>
+      
+        <div class="row-fluid">
+            <div id ="options-select" class="span4">
+                <div id="select-list"></div>
+            </div>
+            <div id="" class="span8">
+                <div id="chart" class='with-3d-shadow with-transitions'>
+                    <svg></svg>
+                </div>
+            </div>
+           
+        </div>
+    </div> 
+{% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+
+  <script>
+    var resource_url = "{{ h.view_resource_url(resource_view, resource, package) }}";
+ 
+    var resource ={{ h.literal(h.dump_json(resource)) }};
+  </script>
+ 
+
```

Comment:
The resource config file doesn't take minified files due to a naming convention problem. We can add the d3 but we need to rename the file. Apparently, it gives out an error if we a filename has multiple ".". Example - d3.v4.min.js